### ISSUE TITLE
feat[design]: ✨ Research on Notification Component usage and inline alternatives

### ISSUE THREAD

**Issue by yoezlem** [solid-design-system/solid#1693]
## Research Ticket
**Investigate the current usage and content of the notification component:**
- [x] How is this component used in practice?
- [x] What types of content are typically placed inside?
- [ ] Could the functionality and requirements be fully addressed with pure inline notifications instead of toasts?

### Examples

https://www.radix-ui.com/primitives/docs/components/toast
https://carbondesignsystem.com/components/notification/accessibility/
https://spectrum.adobe.com/page/toast/ 
beta https://react-spectrum.adobe.com/react-aria/useToast.html#usetoast
https://www.radix-vue.com/components/toast
https://ui.shadcn.com/docs/components/toast

### Technical Information
related to: #1482 

## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Brand approval (if neccessary)
- [x] Figma documentation file incl. hand-off has been created/updated (if applicable)
- [ ] Tokens have been applied/updated (if applicable)
- [ ] Crosslinks are created/updated
- [ ] Implementation works successfully on `figma` branch

**Comment by karlbaumhauer** [comment]
@coraliefeil is handing over this to internal a11y specialist (UI-initiative) in order to get more input on possible solutions.

**Comment by MartaPintoTeixeira** [comment]
Blocked - waiting for input from Damares.

**Comment by coraliefeil** [comment]
@MartaPintoTeixeira @miguelxnxn 

See [Figma Docs](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2265-6175&t=3Nt9xdos6581ardA-11)

After an alignment with Damaris and more research on the topic of how to make toasts accessible I reworked this page and would like to ask you to review it, pls.

Now it should be clear when to use an inline notification and when to use toast notification. Also what needs to be implemented to support accessibility should be well explained.

What do you think? Pls comment in Figma.

Thanks a lot!

**Comment by miguelxnxn** [comment]
@coraliefeil @MartaPintoTeixeira feedback provided on Figma. Let me know If you agree with my points.

**Comment by MartaPintoTeixeira** [comment]
@coraliefeil @miguelxnxn please find my feedback in Figma.

**Comment by coraliefeil** [comment]
Addressed feedback from @miguelxnxn @MartaPintoTeixeira 
Waiting for Damaris comments.

**Comment by coraliefeil** [comment]
[Figma Docs are updated](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2265-6175&t=wyJf0n9rE8Ew5Ko8-11) including feedback from Marta, Miguel and Damaris. The accessibility section now communicates needs to be done to make a toast notification accessible.

How is this component used in practice and what types of content are typically placed inside is answered in Figma Docs.

Could the functionality and requirements be fully addressed with pure inline notifications instead of toasts?
-> No, since inline and toast serve different use cases.

**Comment by coraliefeil** [comment]
@yoezlem research is done. Pls review my last comment and if you agree we can move forward with a11y adjustment for sd-notification.

**Comment by yoezlem** [comment]
Agreed!
